.github/workflows/run_maven_tests.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     - name: Copy ors test config for API tests
       run: cp ./openrouteservice-api-tests/conf/ors-config-test.json ./openrouteservice/src/main/resources/ors-config.json
     - name: Test, build, run API tests
-      run: mvn -B -f openrouteservice/pom.xml verify jacoco:report
+      run: export MAVEN_OPTS="-Xmx3000m -Xms2500m" && mvn -B -f openrouteservice/pom.xml verify jacoco:report
```

Comment:
We should probably use a smaller test set?

Reply:
done